src/sentry/utils/performance_issues/README.md

```diff
@@ -0,0 +1,70 @@
+# Performance Issues (Updated: 2025-05-07)
+
+[Performance issues](https://docs.sentry.io/product/issues/issue-details/performance-issues/) are composed of a few different parts which control how they manifest in the app to users. This document will outline those parts, and then explain how to go about creating a new performance detector, and experimenting with changes to performance detectors.
```

Comment:
is there a reason we are using a README instead of our developer docs?

Reply:
That's a good call out, I decided to keep it in code since this is likely to change or add/remove steps as the team make changes or ACI work begins converting some of these. Keeping it here will let us make changes as we go alongside PRs which is kinda convenient.

The other part is that there are a lot of links to files here which are super handy when previewing in an IDE, rather than hard coding links to `master` branch files on GitHub which someone will have to then find in their IDE.

I do think that once these are somewhat finalized, moving them to develop is the best approach though 🔥